mkdocs.yml

```diff
@@ -78,8 +78,8 @@ nav:
             - 'Control library': 'docs/controls/index.md'
             - 'Configuring controls': 'docs/frameworks-and-controls/configuring-controls.md'
         - Guides:
-            - 'Configure checks on a GitHub repository': 'docs/use-cases/kubescape-gha/kubescape-gha.md'
-            - 'Harden a cluster': 'docs/use-cases/kubescape-cli/kubescape-cli.md'
+            - 'Configure checks on a GitHub repository': 'docs/use-cases/kubescape-gha/index.md'
```

Comment:
I think you also need to rename this file, which doesn't look like it was done in this commit.

Reply:
I updated both with a push several hours ago. I just synced the fork again to be sure. It looks right on my end.